lib/src/controls/inputs/buttons/button.dart

```diff
@@ -40,14 +40,14 @@ class Button extends BaseButton {
         return 0.3;
       }),
       shadowColor: ButtonState.all(theme.shadowColor),
-      padding: ButtonState.all(const EdgeInsets.symmetric(
-        horizontal: 10.0,
-        vertical: 6.0,
-      )),
+      padding: ButtonState.all(const EdgeInsets.only(
+          left: 11.0, top: 5.0, right: 11.0, bottom: 6.0,)),
```

Comment:
Don't forget reformat.

Ps: you can click on "Resolved" on previous issue to indicate that is finished.
For example, when I apply requested modification, I click resolve when I corrected (before commit). With that, I now where I am.